spec/models/ability_spec.rb

```diff
@@ -24,8 +24,8 @@
   context ': an admin set manager' do
     let(:current_user) { FactoryBot.create(:user) }
     let(:admin_set_id) { 'as_mu' }
-    let!(:admin_set) { create(:admin_set, id: admin_set_id, with_permission_template: true) }
-    let!(:admin_set_other) { create(:admin_set, id: batch_other_managed.admin_set_id, with_permission_template: true) }
+    let(:admin_set) { create(:admin_set, id: admin_set_id, with_permission_template: true) }
+    let(:admin_set_other) { create(:admin_set, id: batch_other_managed.admin_set_id, with_permission_template: true) }
 
```

Comment:
So it worked for you without "!". I also tried without it but got error like "permission template not found" with some of the tests.

Reply:
Maybe some of my other changes caused these to be evaluated during the setup without needing to have the `!`.   I removed them after Rubocop complained and was lucky that they weren't needed.